project_wsjf/__manifest__.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Kmee Informática LTDA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Project Wsjf',
+    'summary': """
+        This module implements the Weighted Shortest Job First prioritization model.
```

Comment:
Can u please move the summary to the README.rst and fill the other questions like, Installation, Configure, Contributors and etc.. ?

And check the travis erros.

Reply:
Summay should stay here. Description is to the replaced by the readme.